merlin/dag/dictarray.py

```diff
@@ -0,0 +1,121 @@
+#
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import Dict, Optional
+
+import numpy as np
+
+from merlin.core.protocols import SeriesLike, Transformable
+
+
+class Column(SeriesLike):
```

Comment:
In theory, this is where we might be able to put a shim in between the multiple arrays that represent a column and the code that processes them to make them look more like a single column. TBD what that would look like, but this creates a home for it.